### DISCUSSION TITLE
Get the network adapter and disk details from library template

### DISCUSSION THREAD

**Discussion by pangdanyep** [vmware-archive/vsphere-automation-sdk-python#427] [Q&A] [answered]
How do I get the **network adapter** details of vsphere library ovf template and the **hard disk information** included in the template?
Please, somebody help me!

**Comment by kunalpmj** [comment, marked as answer]
@pangdanyep 

Content Library does not have API to get complete OVF template item information. 
You can use this API : https://developer.broadcom.com/xapis/vsphere-automation-api/latest/vcenter/api/vcenter/ovf/library-item/ovf_library_item_id__action=filter/post/
which return OVF Item Summary. : https://developer.broadcom.com/xapis/vsphere-automation-api/latest/vcenter/data-structures/Ovf_LibraryItem_OvfSummary/
On the other hand VM templates information can be fetched using : https://developer.broadcom.com/xapis/vsphere-automation-api/latest/vcenter/api/vcenter/vm-template/library-items/template_library_item/get/

**Reply by pangdanyep** [reply]
Thank you~ :)
